CCVA_Report.Rmd

```diff
@@ -34,7 +37,7 @@ park <- "BRCA"
 
 
 # read in data
-load(paste0(path, "/", park, "_report_data.RData"))
+load(paste0(path, "/", park, "/", park, "_report_data.RData"))
```

Comment:
I think this is also from me just setting up the bare bones of the report, wasn't sure yet where everything was going to be saved and what it will be named etc.

Reply:
Yeah -- I also figured that was in flux so I didn't update it yet. I also didn't explicit set which variables to save - which I will do once we've select which ones are going in the report.